department-of-veterans-affairs/caseflow
# Intake | Do not allow empty file number in Veteran search

Resolves #9545

### Description
This pr disables search button when it is empty.

### Acceptance Criteria
- [ ] See https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/3794/

## Technical Notes

Would be nice to disallow this on the client side and controller both. Saves a BGS call.
### Testing Plan
1. Go to Intake/search
2. Verify if the button is disabled when the search bar is empty.

<img width="1149" alt="screen shot 2019-02-26 at 12 24 56 pm" src="https://user-images.githubusercontent.com/23080951/53432998-94d57800-39c1-11e9-9836-c03368cb4b85.png">
